src/app/shell/personal-cabinet/provider/create-achievement/create-achievement.component.html

```diff
@@ -22,30 +22,33 @@ <h3 class="wrapper-title">ДОДАТИ ДОСЯГНЕННЯ</h3>
         </app-validation-hint>
 
         <label class="step-label">Учасники<span class="step-required">*</span></label>
-        <mat-form-field appearance="none">
-          <mat-select multiple disableOptionCentering formControlName="childrenIDs" 
-            panelClass="dropdown-panel" class="step-input">
-            <mat-select-trigger>
-              <mat-chip-list>
-                <mat-chip 
-                  *ngFor="let child of AchievementFormGroup.controls['childrenIDs'].value"
-                  [removable]="true" (removed)="onRemoveItem(child, 'childrenIDs')">
-                  {{child.lastName}} {{child.firstName}} {{child.middleName}}
-                  <mat-icon matChipRemove>cancel</mat-icon>
-                </mat-chip>
-              </mat-chip-list>
-            </mat-select-trigger>
-            <mat-option *ngFor="let child of children$"   
-              [value]="child" class="dropdown-option">
-              {{child.lastName}} {{child.firstName}} {{child.middleName}}
-            </mat-option>
-          </mat-select>
-        </mat-form-field>
-        <app-validation-hint 
-          [validationFormControl]="AchievementFormGroup.get('childrenIDs')"
-          [isTouched]="AchievementFormGroup.get('childrenIDs').touched"
-        >
-        </app-validation-hint>
+
+        <ng-container *ngIf="approvedChildren.entities.length !== 0; else noChildren"> 
+          <mat-form-field appearance="none">
+            <mat-select multiple disableOptionCentering formControlName="childrenIDs" 
+              panelClass="dropdown-panel" class="step-input">
+              <mat-select-trigger>
+                <mat-chip-list>
+                  <mat-chip 
+                    *ngFor="let child of AchievementFormGroup.controls['childrenIDs'].value"
+                    [removable]="true" (removed)="onRemoveItem(child, 'childrenIDs')">
+                    {{child.lastName}} {{child.firstName}} {{child.middleName}}
+                    <mat-icon matChipRemove>cancel</mat-icon>
```

Comment:
I haven't noticed that you applied the wrong approach to get control. Please, instead of AchievementFormGroup.controls['childrenIDs'] use the existing method for such goal AchievementFormGroup.get('childrenIDs'). replace it everywhere